<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generalPrintPage</class>
 <widget class="GeneralPrintPage" name="generalPrintPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="typeNumber">
       <property name="text">
        <string>TOBEFILLED:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="number">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="typeDate">
       <property name="text">
        <string>ToBeFILLED</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="date">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kunde:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="salutation">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="street">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="place">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Druck:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Kurz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="positionsShort">
         <property name="text">
          <string>Posten (1)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="groupsShort">
         <property name="text">
          <string>Gruppen (3)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="allShort">
         <property name="text">
          <string>Alles (5)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ausf√ºhrlich</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="positionsLong">
         <property name="text">
          <string>Posten (2)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="groupsLong">
         <property name="text">
          <string>Gruppen (4)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="allLong">
         <property name="text">
          <string>Alles (6)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralPrintPage</class>
   <extends>QDialog</extends>
   <header>pages/general_print_page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
